lib/actions-oclif/scan.ts

```diff
@@ -65,11 +65,11 @@ export default class ScanCmd extends Command {
 		const { discover } = await import('balena-sync');
 		const prettyjson = await import('prettyjson');
 		const { ExpectedError } = await import('../errors');
-		const { dockerPort, dockerTimeout } = await import(
-			'../actions/local/common'
-		);
 		const dockerUtils = await import('../utils/docker');
 
+		const dockerPort = 2375;
+		const dockerTimeout = 2000;
+
```

Comment:
This was the only thing used from `actions/local/common` (any only used here).  So moved here and deleted common.